server/src/librelay/relay_server_test.go

```diff
@@ -128,6 +129,8 @@ var client *TestClient
 var relayKey1 *ecdsa.PrivateKey
 var gaslessKey2 *ecdsa.PrivateKey
 var ownerKey3 *ecdsa.PrivateKey
+var unstakeDelay = big.NewInt(1 * 60 * 60 * 24 * 7) // 1 week in seconds
```

Comment:
it bothers me to see these constants... did we ever test that a relay works with DIFFERENT values for stake delay/amount ?

Reply:
It looks like we do in `RelayHub.test.js`